core/gather/gatherers/full-page-screenshot.js

```diff
@@ -31,8 +31,8 @@ function getObservedDeviceMetrics() {
   // Convert the Web API's kebab case (landscape-primary) to camel case (landscapePrimary).
   const screenOrientationType = kebabCaseToCamelCase(window.screen.orientation.type);
   return {
-    width: document.documentElement.clientWidth,
-    height: document.documentElement.clientHeight,
+    width: window.outerWidth,
+    height: window.outerHeight,
```

Comment:
Creating the smoke test helped me discover that `document.documentElement.client*` does not represent the actual device size.